samples/boards/stm32/power_mgmt/blinky/sample.yaml

```diff
@@ -10,5 +10,6 @@ tests:
         - "Device ready"
     filter: dt_compat_enabled("zephyr,power-state") and
             dt_enabled_alias_with_parent_compat("led0", "gpio-leds") and
-            dt_compat_enabled("st,stm32-lptim")
+            dt_compat_enabled("st,stm32-lptim") and
+            dt_chosen("zephyr,pm-timer")
```

Comment:
This PR's usage of this chosen node is not acceptable right now.

1. Zephyr chosen nodes are for configuring a piece of hardware used by a subsystem, like the console. This PR seems to introduce a new chosen node that doesn't satisfy this requirement. The commit logs provide no justifications.
2. All such chosen nodes are documented in the table at the bottom of doc/build/dts/api/api.rst. If you can convince me that the requirements for 1. are met, we will need this.

Reply:
> This PR's usage of this chosen node is not acceptable right now.
> 
> 1. Zephyr chosen nodes are for configuring a piece of hardware used by a subsystem, like the console. This PR seems to introduce a new chosen node that doesn't satisfy this requirement. The commit logs provide no justifications.
> 2. All such chosen nodes are documented in the table at the bottom of doc/build/dts/api/api.rst. If you can convince me that the requirements for 1. are met, we will need this.

1- the chosen node for the `pm-timer` is placed to select which HW peripheral will be used for power-management timer. Some stm32 MCUs have more than one instance of LPTIM. Even if several LPTIM devices are enabled in the DTS, only one device is selected for PM timer : the _chosen_ one.